### PR TITLE
Fix right icon in full screen.

### PR DESCRIPTION
**Pull request by codewithvk**
* Resolves: check not right
* Target version: master 

### Summary

I had added right icon when we are on full screen.

### Checklist

- [x] Code is properly formatted
- [x] All commits have Change-Id
- [x] I have run tests with `make check`
- [x] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

https://user-images.githubusercontent.com/61119120/115405981-5c7dfd00-a20c-11eb-8b53-27ea5a5bf11b.mp4

### CODE REVIEW

**Comment by codewithvk** [comment]
@lpranam

File: loleaflet/src/control/Control.UIManager.js [hunk 1]
```diff
@@ -404,6 +404,15 @@ L.Control.UIManager = L.Control.extend({
 		return $('.loleaflet-ruler').is(':visible');
 	},
 
+	// fullscreen
+
+	isFullscreen: function() {
+		if (document.fullscreenElement != null)
```
Comment (lpranam) [inline loleaflet/src/control/Control.UIManager.js:410]: you have to add conditions for all the browsers like done here:
https://github.com/CollaboraOnline/online/blob/465c435e6863a377611ccff2ef082b37c0967ba1/loleaflet/src/core/Util.js#L166

File: loleaflet/src/control/Control.UIManager.js [hunk 2]
```diff
@@ -404,6 +404,15 @@ L.Control.UIManager = L.Control.extend({
 		return $('.loleaflet-ruler').is(':visible');
 	},
 
+	// fullscreen
```
Comment (lpranam) [inline loleaflet/src/control/Control.UIManager.js:407]: no need for this comment

**Review by lpranam — CHANGES_REQUESTED** [review]

File: loleaflet/src/control/Control.UIManager.js (hunk 1 above)
Comment (codewithvk) [reply loleaflet/src/control/Control.UIManager.js]: I did changes, can you re-review it?

Comment (lpranam) [reply loleaflet/src/control/Control.UIManager.js]: I can't see other conditions.

**Comment by lpranam** [comment]
you also need to remove all the other non relevant and unnecessary changes from the PR

**Comment by lpranam** [comment]
1. please have some meaningful commit message
2. I still can't see all the conditions I have pointed to you earlier.

**Pull request closed without merging**